docs/docs/quickstart.mdx

```diff
@@ -171,203 +143,260 @@ healthy system you're ready to roll! A healthy system will show something along
 
-As already explained, we're proxying requests to ORY Kratos' Public API. We are
-doing this because that way all requests are going to and coming from the same
-hostname. This avoids common cross-domain issues with cookies.
+In order to avoid common cross-domain issues with cookies, we're proxying requests to Kratos' Public API so that all requests come from the same
+hostname.
 
 ## Perform registration, logout, login
 
-Enough theory, it's time to get this thing going! Let's start by trying to open
-the dashboard - **go to
-[127.0.0.1:4455/dashboard](http://127.0.0.1:4455/dashboard)**. You will probably
-notice that you're ending up at the login endpoint:
+Enough theory! Let's start by trying to open
+the dashboard: go to [127.0.0.1:4455/dashboard](http://127.0.0.1:4455/dashboard).
+
+You should notice that you're ending up at the login endpoint instead of the dashboard:
 
-<img
-  alt="Login screen of your secured app"
-  src={useBaseUrl('img/docs/secureapp-login.png')}
-/>
+<p>
```

Comment:
Markdown should wrap this automatically in a paragraph, no?

Reply:
It wasn't spacing things properly for me. The previous screenshots had some transparent padding in the PNGs themselves, but the ones I updated them to don't.

Reply:
Interesting, then we'll use the paragraph!